src/lib/objects.js

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable object-shorthand */
+const createPerson = (name, age) => {
+  return {
+    name: name,
+    // eslint-disable-next-line object-shorthand
+    age: age
+  };
+};
```

Comment:
These eslint exceptions are not needed, you should remove these. Just for reference, eslint is not something you need to fight with as it is trying to help you (though it may not feel like it sometimes). Once you format the file after removing these it will correct these things for you